portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,52 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sharon Dong</title>
     <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Work Sans' rel='stylesheet'>    
     <script src="script.js"></script>
   </head>
+  <div class="topnav">
+    <div class="topnav-left">
+      <a href="index.html">Sharon Dong</a>
+    </div>
+    <a href="#contact">Contact</a>
+    <a href="pics.html">Fun Photos</a>
+    <a href="projects.html">Projects</a>
+    <a href="#about-me">About</a>
+  </div>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="cover"></div>
```

Comment:
Is this div doing anything?

Reply:
It's actually for the blurry cover photo. All the code for it is in #cover class in the css file.